### ISSUE TITLE
Trouble with exec

### ISSUE THREAD

**Issue by coderaiser** [nodejitsu/forza#36]
Good Morning.

Could you please help me with one problem?
When I call function `require('child_process').exec` by my [application](http://cloudcmd.io) on [jitsu](http://jit.su) service I got this error on my logs:

```
ld.so.1: ksh93: fatal: /opt/local/lib/node_modules/forza/libinterposed.so: wrong ELF class: ELFCLASS64
```

And nothing is executed. Version of Forza on **jitsu** is **0.2.0**.

Thank you.

**Comment by jcrugzz** [comment]
@coderaiser is this still occurring?

**Comment by coderaiser** [comment]
No, this trouble has disappeared on jitsu somehow,  thank you.

**Comment by jcrugzz** [comment]
fixed in 002572c3beec6508f3f2e469107fa7bb68142687
